### PR TITLE
Workflow

### PR DESCRIPTION
**Pull request by abuzarmahmood**
Automatically testing spike sorting and emg pipelines using test dataset

### CODE REVIEW

**Review by abuzarmahmood — COMMENTED** [review]
Only significant change is taking out additional analyses from blech_palatability_identity_setup

File: params/env_params.json
Comment (hfgem) [inline params/env_params.json:1]: Probably shouldn't be specific to your machine?

File: requirements/conda_requirements_base.sh
Comment (hfgem) [inline requirements/conda_requirements_base.sh:1]: you don't need umap anymore?

File: params/env_params.json
Comment (hfgem) [inline params/env_params.json:1]: This probably shouldn't be specific to your machine?

File: requirements/conda_requirements_base.sh
Comment (abuzarmahmood) [reply requirements/conda_requirements_base.sh:1]: It got offloaded to neuRecommend which now has pretrained versions...we can keep it but it usually takes too long to train for each new channel

File: params/env_params.json
Comment (abuzarmahmood) [reply params/env_params.json:1]: Good catch! I mean to put up setup instructions in the readme for the (now 3) params files that can/need to be edited. So this string will need to be replaced...

Comment (abuzarmahmood) [reply params/env_params.json:1]: As it is, is also will only affect anyone running the workflow test...so it won't mess with regular usage

**Review by hfgem — APPROVED** [review]

**Pull request merged**